pkg/ansible/watches/watches_test.go

```diff
@@ -68,10 +69,10 @@ func TestNew(t *testing.T) {
 			}
 			if watch.ReconcilePeriod != expectedReconcilePeriod {
 				t.Fatalf("Unexpected reconcilePeriod %v expected %v", watch.ReconcilePeriod,
-					expectedReconcilePeriod)
+					reconcilePeriodDefault)
```

Comment:
The `expectedReconcilePeriod` is above so this change shows not required at all.